### PR TITLE
Responsive tweaks and refactor for homepage styles

### PR DESCRIPTION
**Pull request by glynnis**
Connects #7.

Still left TODO (i.e. not addressed by this PR):
* Image compression
* Responsive tweaks/fallout on about and contact pages
* Complete refactor/removal of old breakpoints from `_base.scss`

My browser extension for full-page screen caps didn't elegantly interpret the JS/nav switchover, so these aren't a totally accurate reflection of how things look (but it looks fine during normal usage). Probably easiest to check out locally to take a look:

## Small
screencapture-localhost-3000-index-html-1510280620240

## Medium
screencapture-localhost-3000-index-html-1510280584381

## Large
screencapture-localhost-3000-index-html-1510280485217

### CODE REVIEW

**Comment by glynnis** [comment]
I've added commits to this PR that include the responsive tweaks needed for the contact and about pages, removing the last of the old breakpoint code that used `max-width`. I've got the hang of switching them over, so as I think I mentioned elsewhere, I'm happy to help make the switch if you're still using any of those on other branches.

## Small
screencapture-localhost-3000-contact-html-1510339818279

screencapture-localhost-3000-about-html-1510339738900

## Medium
screencapture-localhost-3000-contact-html-1510339807208

screencapture-localhost-3000-about-html-1510339760849

## Large
screencapture-localhost-3000-contact-html-1510339791218

screencapture-localhost-3000-about-html-1510339775853

**Comment by glynnis** [comment]
Connects #40

File: app/index.html [hunk 1]
```diff
@@ -97,21 +97,23 @@ <h1 class="uppercase section-title">What we can do for you</h1>
...
+          <p>We help connect you with your audience and get results. From branding to design systems, user
+            research to interfaces, we've got you covered.</p>
           <p>We create consistent, accessible, and stunning user experiences.</p>
         </div>
       </div>
 
       <div class="service">
-        <img src="./images/mobile.gif" alt="Mobile phone with paper mockups">
-        <div class="service-description ">
+        <img src="./images/consulting.jpg" alt="Four people fist-pumping over a
```
Comment (siakaramalegos) [inline app/index.html]: I believe this is fist-bumping not pumping. LOL.

**Comment by glynnis** [comment]
Connects #40

**Pull request merged**